api/src/main/java/com/github/eyefloaters/console/api/support/SizeLimitedSortedSet.java

```diff
@@ -0,0 +1,47 @@
+package com.github.eyefloaters.console.api.support;
+
+import java.util.Comparator;
+import java.util.TreeSet;
+
+/**
+ * A SortedSet that is limited to a fixed size. Records beyond the size limit
+ * will be removed immediately. This class is useful for building paged data
```

Comment:
You should mention the policy by which records are removed.

Reply:
Refined the JavaDoc for the `SizeLimitedSortedSet`:

https://github.com/eyefloaters/console/blob/40b13c2b2b91e91eecf1bd543b9346d55b9abe76/api/src/main/java/com/github/eyefloaters/console/api/support/SizeLimitedSortedSet.java#L7-L22